rom/dev/src/fht.rs

```diff
@@ -183,14 +183,14 @@ pub fn make_fht(env: &RomEnv) -> FirmwareHandoffTable {
     }
 }
 
-pub fn load_fht(fht: FirmwareHandoffTable) {
+pub fn store_fht(fht: FirmwareHandoffTable) {
```

Comment:
Can we use this ? https://github.com/chipsalliance/caliptra-sw/blob/main/common/src/hand_off.rs#L423

Reply:
I still want to use "load" and "store". Easy to understand. I will change the name to "store".